lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -44,52 +45,51 @@ function computeDescription(ast, property, startRange) {
 
 /**
  * @param {string} dir
- * @param {Record<string, ICUMessageDefn>} strings
+ * @return {Record<string, ICUMessageDefn>}
```

Comment:
can you revert the changes to the param/return type (here and for `createPsuedoLocaleStrings`) so it's only an internal change? the changes are fine, but it's going to make updating #9114 that much more work after this lands

Reply:
let's just chill this until after placeholders land